app/templates/sticky_footer.hbs

```diff
@@ -22,7 +22,7 @@
             <i>{{fa-icon 'cube'}}</i>{{t "index.items"}}
           {{/link-to}}
 
-          <li><i>{{fa-icon 'qrcode'}}</i>{{t "index.goto"}}</li>
+          {{!-- <li><i>{{fa-icon 'qrcode'}}</i>{{t "index.goto"}}</li> --}}
```

Comment:
Why have we commented out this line ?

Reply:
Because it doesn't redirect anywhere

Reply:
oh, okkay. So Is that's why the ios-build was failing ?

Reply:
No, that's why they rejected our app on iTunes

Reply:
Okay great.. excellent work 👍